package/netsnmp/0003-fix-curfilename-not-static.patch

```diff
@@ -0,0 +1,11 @@
+--- netsnmp-5.9_orig/snmplib/read_config.c.origin	2022-05-29 18:30:48.190090824 +0300
++++ netsnmp-5.9_new/snmplib/read_config.c	2022-05-29 18:32:15.668226606 +0300
+@@ -473,7 +473,7 @@
+ #endif
+ 
+ static unsigned int  linecount;
+-static const char   *curfilename;
++const char   *curfilename;
```

Comment:
This variable is referenced by the Siklu SNMP plugin, so it can't be static. In Net-SNMP 5.4.2.1, it wasn't static.